src/opm/parser/eclipse/EclipseState/Schedule/Well/Well.cpp

```diff
@@ -1406,6 +1415,9 @@ int Opm::eclipseControlMode(const Opm::Well::ProducerCMode pmode,
     using PMode = ::Opm::Well::ProducerCMode;
     using Val   = ::Opm::RestartIO::Helpers::VectorItems::IWell::Value::WellCtrlMode;
 
+    if (wellStatus == ::Opm::Well::Status::SHUT) {
+        return Val::Shut;
+    }
```

Comment:
This too requires updating [`WellTests`](https://github.com/OPM/opm-common/blob/269537b1015aedd904e55dc26ba18091bf25bf1d/tests/parser/WellTests.cpp#L1038).